### PR TITLE
Set LiveView release to Shelley

### PR DESCRIPTION
**Pull request by gufmar**

### CODE REVIEW

**Comment by dcoutts** [comment]
It should really take this from the protocol being run rather than being hard coded.

**Comment by gufmar** [comment]
Agree. Might with a moving slider between 0/1 of Shelley blocks.

I just wanted to do a favour to the comms people, when all the screenshots are posted.

File: cardano-node/src/Cardano/Node/TUI/LiveView.hs [hunk 1]
```diff
@@ -492,7 +492,7 @@ initLiveViewState = do
 
     return $ LiveViewState
                 { lvsScreen                 = MainView
-                , lvsRelease                = "Byron"
+                , lvsRelease                = "Shelley"
```
Comment (erikd) [inline cardano-node/src/Cardano/Node/TUI/LiveView.hs:495]: This change would also affect the node running on the mainnet which is still `Byron`.

Either this should stay as is until we actually do switch mainnet to Shelley of it should get the correct value for that from somewhere further up the software stack.

**Comment by erikd** [comment]
I have a work in progress fix for this that correctly gets this value from the configuration.

**Comment by Jimbo4350** [comment]
Closed in favour of #974

**Pull request closed without merging**